src/app/app.component.ts

```diff
@@ -1,10 +1,15 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
+  word: string = '';
```

Comment:
```suggestion
  word = '';
```
when assign a variable is not necessary to declare its type (its infer from the assignation)